packages/wdio-allure-reporter/src/index.js

```diff
@@ -40,23 +42,50 @@ class AllureReporter extends WDIOReporter {
     }
 
     onSuiteStart(suite) {
-        const currentSuite = this.allure.getCurrentSuite()
-        const prefix = currentSuite ? currentSuite.name + ': ' : ''
-        this.allure.startSuite(prefix + suite.title)
+        if (this.options.useCucumberStepReporter) {
```

Comment:
How about this:
```js

onSuiteStart(suite) {
    if (this.options.useCucumberStepReporter) {
        if (suite.type === 'feature') {
            // handle cucumber features as allure "suite"
            return this.allure.startSuite(suite.title)
        }
        
        // handle cucumber scenarii as allure "case" instead of "suite"
        this.allure.startCase(suite.title)
        return this.setCaseParameters(suite.cid)
    }

    const currentSuite = this.allure.getCurrentSuite()
    const prefix = currentSuite ? currentSuite.name + ': ' : ''
    this.allure.startSuite(prefix + suite.title)
}
```